Solves an issue where running 'feat' produces the error
"can't read "::errorCode": no such variable"
See issue and solution at
https://www.jiscmail.ac.uk/cgi-bin/webadmin?A2=ind1412&L=FSL&D=0&P=300830
Author: Caspar van Leeuwen (SURFsara)
diff -Nru fsl.orig/src/misc_tcl/fsl_exec.tcl fsl/src/misc_tcl/fsl_exec.tcl
--- fsl.orig/src/misc_tcl/fsl_exec.tcl	2019-10-21 15:53:58.647058588 +0200
+++ fsl/src/misc_tcl/fsl_exec.tcl	2019-10-21 16:28:52.811113241 +0200
@@ -175,6 +175,7 @@
 	set logout ""
     }
 
+    if { ! [ info exists ::errorCode ] } { set ::errorCode "NONE" }
     # run and log the actual command
     if { $do_logout } {
 	fsl:echo $logout "\n$thecommand"
